app/src/main/java/com/example/se1_hw/SignInActivity.kt

```diff
@@ -0,0 +1,86 @@
+package com.example.se1_hw
+
+import android.app.Activity
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import com.example.se1_hw.databinding.ActivityMainBinding
+
+class SignInActivity : AppCompatActivity() {
+    lateinit var binding:ActivityMainBinding
+    private val REQUEST_CODE = 100
```

Comment:
```suggestion
   companion object {
        private const val SIGN_UP_RESULT_CODE = 100
    }
```

- 이 부분도 잘해주었는데, KOTLIN에서 상수 값은 클래스 마지막 부분에 위에 내가 적어 놓은 companion object 를 사용해서 적어주는 코드 컨벤션이 있어 !! 다른 사람들이랑 협업을 할 때는 이렇게 써주는게 서로에 대한 약속 (!?) ,, ㅎㅎ 

- 조금 더 설명하자면 코틀린 코드 작성 규칙은 이런 식으로 정렬된다고 해 !! 지금 알아두면 나중에 코드 짤 때 도움 될 것 같아서 한번 읽어봐바 😍

1. 속성 선언 및 초기화 블록 

2. 보조 생성자 

3.  함수 선언

4. Companion Object 

이 순서대로 !!!! 그리고 알파벳순이나 가독성으로 메소드 선언을 해주는 것보다, 기능적으로 연관되어있는 메서드끼리, 위에서 아래방향으로 읽을 때, 논리적으로 이해할 수 있도록 !!! 아직 처음이라 어색하겠지만 !!! 그리고 나도 아직 이런거 지키지 못하는데, 이번 기회로 우리 같이 이렇게 해보자 !! 화이팅 !!!

Reply:
아하!! 좋은 정보 감사합니당~!!ㅎㅎ